apps/game_backend/lib/game_backend/quests/quest_description.ex

```diff
@@ -0,0 +1,34 @@
+defmodule GameBackend.Quests.QuestDescription do
+  @moduledoc """
+
+  """
+
+  use GameBackend.Schema
+  import Ecto.Changeset
+
+  schema "quest_descriptions" do
+    field(:description, :string)
+    field(:type, :string)
+    field(:quest_objectives, {:array, :map})
+
+    timestamps()
+  end
+
+  @types ["daily"]
```

Comment:
Unless we actually use this outside the module don't do this and put it directly in `validate_inclusion/3`

Reply:
not used at all, removed